src/fswAlgorithms/attGuidance/sunSafePointCpp/sunSafePointCpp.h

```diff
@@ -39,20 +39,35 @@ class SunSafePointCpp: public SysModel {
     void UpdateState(uint64_t CurrentSimNanos) override;        //!< Update member function
 
-    double minUnitMag;                                          //!< The minimally acceptable norm of sun body vector
-    double sunAngleErr;                                         //!< [rad] The current error between cmd and obs sun angle
-    double smallAngle;                                          //!< [rad] An angle value that specifies what is near 0 or 180 degrees
-    Eigen::Vector3d eHat180_B;                                  //!< Eigen axis to use if commanded axis is 180 from sun axis
-    Eigen::Vector3d sunMnvrVec;                                 //!< The Eigen axis that we want to rotate on to get sun
-    Eigen::Vector3d sHatBdyCmd;                                 //!< Desired body vector to point at the sun
-    Eigen::Vector3d omega_RN_B;                                 //!< Desired body rate vector if no sun direction is available
-    double sunAxisSpinRate;                                     //!< [rad/s] Desired constant spin rate about sun heading vector
+    double getMinUnitMag();                                     //!< Getter method for the minimally accepted sun body vector norm
+    double getSmallAngle();                                     //!< Getter method for the small alignment tolerance angle near 0 or 180 degrees
+    double getSunAxisSpinRate();                                //!< Getter method for the desired constant spin rate about sun heading vector
+    const Eigen::Vector3d &getOmega_RN_B() const;               //!< Getter method for the desired body rate vector if no sun direction is available
+    const Eigen::Vector3d &getSHatBdyCmd() const;               //!< Getter method for the desired body vector to point at the sun
+    void setMinUnitMag(double minUnitMag);                      //!< Setter method for the minimally accepted sun body vector norm
+    void setSmallAngle(double smallAngle);                      //!< Setter method for the small alignment tolerance angle near 0 or 180 degrees
+    void setSunAxisSpinRate(double sunAxisSpinRate);            //!< Setter method for the desired constant spin rate about sun heading vector
+    void setOmega_RN_B(Eigen::Vector3d &omega_RN_B);            //!< Setter method for the desired body rate vector if no sun direction is available
+    void setSHatBdyCmd(Eigen::Vector3d &sHatBdyCmd);            //!< Setter method for the desired body vector to point at the sun
 
     ReadFunctor<NavAttMsgPayload> imuInMsg;                     //!< IMU attitude guidance input message
     ReadFunctor<NavAttMsgPayload> sunDirectionInMsg;            //!< Sun attitude guidance input message
     Message<AttGuidMsgPayload> attGuidanceOutMsg;               //!< Attitude guidance output message
 
     BSKLogger *bskLogger;                                       //!< BSK Logging
+
+private:
+    // User-configurable variables
```

Comment:
:large_orange_diamond: We can delete this and the comment below. They are redundant because we've now indicated clear what is available to the user on the public API of this module.

Reply:
Done